src/test/java/com/thealgorithms/others/NewManShanksPrimeTest.java

```diff
@@ -0,0 +1,54 @@
+package com.thealgorithms.others;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.thealgorithms.dynamicprogramming.NewManShanksPrime;
+public class NewManShanksPrimeTest {
+    @Test
+	void testOne() 
+	{
+		System.out.println((NewManShanksPrime.nthManShanksPrime(1)));
```

Comment:
Please add assertions to your tests. The tests must validate that the calculated number is what we expect for the corresponding input.